deb/build

```diff
@@ -22,6 +22,38 @@ genchangelog()
     echo " -- $3  `LANG=C date -R`"
 }
 
+versionedlibrary()
+{
+    # package major version
+    pkgsuffix=$1
+
+    curr_dir=$PWD
+
+    # cd to the `make` install path
+    cd $2
+
+    # rename the original library file to the versioned library file
+    mv $pkgname.so.$libversion "$pkgname$pkgsuffix.so.$libversion"
+    mv $pkgname.a "$pkgname$pkgsuffix.a"
+    mv $pkgname.la "$pkgname$pkgsuffix.la"
```

Comment:
Why not doing all this when creating the package via `fpm` mappings instead? It looks overly complicated.